cirq/google/vis/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 The Cirq Developers
```

Comment:
I think we should put this in cirq.vis and not cirq.google.  Cirq.google is for google specific stuff and this isn't really that.  (contradicting @Strilanc from another thread).

Reply:
Done.